bootstrap/util/configowner_test.go

```diff
@@ -20,113 +20,134 @@ import (
 	"context"
 	"testing"
 
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/utils/pointer"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
-func TestGetConfigOwnerSuccess(t *testing.T) {
+func TestGetConfigOwner(t *testing.T) {
+	g := NewGomegaWithT(t)
```

Comment:
```suggestion
	g := NewWithT(t)
```
NewGomegaWithT is deprecated: https://godoc.org/github.com/onsi/gomega#GomegaWithT